.travis.yml

```diff
@@ -63,139 +60,703 @@ jobs:
 
     - <<: *base-test
       stage: test
-      name: "generic, login, component command and plugin handler integration tests"
+      name: "Generic test"
       script:
         - ./scripts/oc-cluster.sh
         - make configure-supported-311-is
         - make bin
         - sudo cp odo /usr/bin
         - odo login -u developer
         - travis_wait make test-generic
+        - odo logout
+
+
+    - <<: *base-test
```

Comment:
Not sure why we are separating the CI jobs again. @cdrage I think we have already discussed it in pr comment https://github.com/openshift/odo/pull/3776#issuecomment-677433898 . 

Is there any other factor you are considering which may be I am not aware of. Why I am saying this because we are always worried about CI timings and separating jobs increasing it to more than an hour.
![image](https://user-images.githubusercontent.com/51620925/92207618-253e2400-eea7-11ea-9b52-aa95919cbb38.png)

Reply:
@prietyc123 Yes, I know. The problem is that we keep having Travis flakes. Most likely because of being out of memory int he job, as we are running too many things at once in the Travis VM's. This travis modification is temporary in order for me to diagnose it easier.

Reply:
I guess there should not be any memory outage because as per the travis documentation [1] the memory allocated to ```dist: xenial``` is ```7.5 GB``` and as part of parallel process is concerned we can run 4 thread max [2] without degrading the travis performance. 

[1] https://docs.travis-ci.com/user/reference/overview/
[2] https://docs.travis-ci.com/user/common-build-problems/#parallel-processes